### ISSUE TITLE
(Maybe) default py=True in launch_fluent

### ISSUE THREAD

**Issue by seanpearsonuk** [ansys/pyfluent#3049]
It might be appropriate at this stage to default the launched instance to Python rather than legacy console.

**Comment by prmukherj** [comment]
I agree with this idea.

**Comment by hpohekar** [comment]
Yes, if we are launching Fluent using PyFluent (`launch_fluent`) then it should be in launch Fluent with Python console.

**Comment by mkundu1** [comment]
Note that, there is a current issue while starting the Fluent container with -py, reported by @raph-luc 
```
Error: libXss.so.1: cannot open shared object file: No such file or directory
Error Object: ("solver = ansys.fluent.core.session_solver.Solver(fluent_connection=ansys.fluent.core.fluent_connection.FluentConnection(channel='', cleanup_on_exit=False, start_transcript=False))")
```
The error is present in both 24.2 and 25.1 versions of the image in all -hidden/-gu/-g mode. Ideally, the library libXss, which is a screensaver library, should not be required.

**Comment by mkundu1** [comment]
> Note that, there is a current issue while starting the Fluent container with -py, reported by @raph-luc
> 
> ```
> Error: libXss.so.1: cannot open shared object file: No such file or directory
> Error Object: ("solver = ansys.fluent.core.session_solver.Solver(fluent_connection=ansys.fluent.core.fluent_connection.FluentConnection(channel='', cleanup_on_exit=False, start_transcript=False))")
> ```
> 
> The error is present in both 24.2 and 25.1 versions of the image in all -hidden/-gu/-g mode. Ideally, the library libXss, which is a screensaver library, should not be required.

This is due to an unintended import in PyConsole code, will merge a fix in Fluent.

**Comment by hpohekar** [comment]
@mkundu1 Thanks a lot for the update. I was confused about why build is getting hang every time for this PR only during codegen. Now it's clear.

**Comment by hpohekar** [comment]
I had also reverted file transfer changes in this PR but still it was hanging every time in CI.

**Comment by seanpearsonuk** [comment]
> > Note that, there is a current issue while starting the Fluent container with -py, reported by @raph-luc
> > ```
> > Error: libXss.so.1: cannot open shared object file: No such file or directory
> > Error Object: ("solver = ansys.fluent.core.session_solver.Solver(fluent_connection=ansys.fluent.core.fluent_connection.FluentConnection(channel='', cleanup_on_exit=False, start_transcript=False))")
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The error is present in both 24.2 and 25.1 versions of the image in all -hidden/-gu/-g mode. Ideally, the library libXss, which is a screensaver library, should not be required.
> 
> This is due to an unintended import in PyConsole code, will merge a fix in Fluent.

Thanks, @mkundu1
